conf/routes

```diff
@@ -11,3 +11,6 @@ GET     /assets/*file               controllers.Assets.versioned(path="/public",
 
 # API endpoints
 GET /pass-bakery/status             controllers.APIController.serviceStatus()
+#GET /rest/bakery/product/:id            controllers.APIController.findID()
```

Comment:
While this may be the future, right now this is commented. Remove if you are not using it